### DISCUSSION TITLE
lasergrbl laser working but not engraving

### DISCUSSION THREAD

**Discussion by bdp137** [arkypita/LaserGRBL#986] [Q&A]
Everything seems to be working and engraving light is going over the board following the picture, but not actually engraving.

**Comment by StuartB4** [comment]
Is it focussed properly? Are you trying to engrave on white material, that's harder to do because it reflects most of the laser power.
What are the power settings and speed settings, if it's too fast and not high enough power it won't work.
Have you managed to engrave anything at all before?

**Comment by bdp137** [comment]
We just got this so we have never engraved before.  We know how to control the speed but are unsure of what the focus is?

**Comment by StuartB4** [comment]
There should be an adjustable ring on the bottom of the laser, screw it in or out to get the smallest dot possible to focus it.
